CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 <!-- There should always be "Unreleased" section at the beginning. -->
 
 ## Unreleased
+- [**BC**] Require php 7.4 and update dependencies
```

Comment:
Requiring new PHP version is not a BC, as you don't (I guess) change the library public API.